packages/affiliates/libs/affiliates/devmining.js

```diff
@@ -87,10 +87,25 @@ module.exports = ({ queries, empAbi, coingecko, synthPrices, firstEmpDate }) =>
     }
   }
 
+  function decodeMedianizerPrice(input) {
+    return [input[0] * 1000, input[1]];
+  }
+  function decodeOtherPrice(input) {
+    return [input.closeTime * 1000, input.closePrice.toString()];
+  }
+  function isMedianizerPrice(input) {
+    return input.length === 2;
+  }
   // Gets the value of synthetics in collateral currency.
   async function getSyntheticPriceHistory(address, start, end) {
     const result = await synthPrices.getHistoricSynthPrices(address, start, end);
-    return Prices(result);
+    // Turns out getHistoricPriceFeed differs between price feed classes. This needs to be fixed
+    // but for now we will detect it and switch between decodings.
+    const fixedPrices = result.map(price => {
+      if (isMedianizerPrice(price)) return decodeMedianizerPrice(price);
+      return decodeOtherPrice(price);
+    });
+    return Prices(fixedPrices);
```

Comment:
Yeah, we should definitely fix the underlying price feed. Can you add a TODO?

Reply:
for reference see #2461. ive also added a todo